pkg/ocicni/ocicni.go

```diff
@@ -658,11 +658,6 @@ func (plugin *cniNetworkPlugin) TearDownPodWithContext(ctx context.Context, podN
 	plugin.podLock(podNetwork).Lock()
 	defer plugin.podUnlock(podNetwork)
 
-	if err := tearDownLoopback(podNetwork.NetNS); err != nil {
-		// ignore error
-		logrus.Warningf("Ignoring error tearing down loopback interface: %v", err)
-	}
```

Comment:
@dcbw do you have more context on that and if it's safe to remove?

Reply:
This is doing effectively (including the ns setup)
Pod Setup:
ip netns add ns0
ip netns ip link set lo up

Pod Tear down:
ip netns ip link set lo down
ip netns del ns0

We are just skipping the 'ip link set lo down' piece

In the end, the network namespace is deleted and no frames are sent between the host/ns with the state of lo changing